README.md

```diff
@@ -1 +1,145 @@
-# angular-http-readme
\ No newline at end of file
+# $http
+
+## Overview
+
+One common requirement of apps is the ability to make HTTP requests. Luckily, Angular provides us a service named `$http` to do these with ease - let's take a look!
+
+## Objectives
+
+- Describe $http
+- Use $http methods
+- Fetch data from an API
+- Update data from an API
+
+## What is $http?
+
+$http is a core Angular service that provides a simplistic API to allow us to communicate with HTTP endpoints with ease.
+
+There are a few ways to do requests - let's take a look at them
+
+### $http()
+
+We can use `$http` as a function, passing through a configuration object with it, as such:
+
+```js
+$http({
+	method: 'GET',
+	url: '/someURL'
+});
+
+$http({
+	method: 'POST',
+	data: {
+		username: 'Bill'
+	},
+	url: '/someOtherURL'
+});
+```
+
+This will return a promise with the data - we call the `.then` function that the function call returns - passing through a callback function that will get called when the request has finished.
```

Comment:
At this point, the students will need more clarification on promise objects. We have another README on promises that we can insert before this one in the curriculum, but it would still be a good idea to include a brief explanation of what it is.